docs/test-results/badges-browser-matrix.md

```diff
@@ -43,12 +45,12 @@ Alternatively, you can use this HTML code on your project site.
 ```
 
 :::tip Multiple Projects, Multiple Accounts
-If you just have one project, you can use your main Sauce account name. If you have multiple projects, you will want to create a sub-account for each project.
+If you just have one project, you can use your main Sauce Labs account name. If you have multiple projects, you should create a sub-account for each project.
 :::
 
 2. Run your tests.
-3. Make sure to set a build number, a pass/fail status, and a public, share, or public restricted visibility for every test that runs using the Test Configuration Options.
-You'll know that these are set correctly if your tests have a status of Pass or Failed instead of Finished on your dashboard, and that a build number is also displayed.
+3. For every test that runs using the test configuration options:<ul><li>Set a build name on your capabilities (see [Use Build IDs, Tags, and Names to Identify Your Tests](https://docs.saucelabs.com/basics/test-config-annotation/test-annotation/#use-build-ids-tags-and-names-to-identify-your-tests)).</li><li>Set pass/fail on your tests via some of our different options (see [Setting Pass/Fail](https://docs.saucelabs.com/basics/test-config-annotation/test-annotation/#setting-passfail)).</li><li>Set test visibility to public (see [Test Configuration Options - public](https://docs.saucelabs.com/dev/test-configuration-options/#public)). If the visibility shouldn't be public, see [Status Images for Private Accounts](https://docs.saucelabs.com/test-results/badges-browser-matrix/#status-images-for-private-accounts).</li></ul>
```

Comment:
I am not sure this will render as expected. Typically, once you introduce HTML, you are not able to themn have markdown elements inside the HTML. So here, where you have an HTML unordered list, the markdown link syntaxt `[linktext](url)` won't display as a link in the rendered site. You need to use an `<a href=>` HTML link reference instead. You can check your local build, I might be wrong, but I'm pretty sure that's how it works.

Reply:
When I checked all the new links in my local build before I submitted the PR, they worked fine. I just checked again and had the same result. I'm hoping there isn't some voodoo magic that happens when I merge ¯\(°_°)/¯